src/libmongoc/doc/mongoc_collection_get_last_error.rst

```diff
@@ -3,6 +3,9 @@
 mongoc_collection_get_last_error()
 ==================================
 
+.. warning::
+   .. deprecated:: 1.9.0
+
```

Comment:
Is there a replacement/recommendation to be documented with the deprecation?

Reply:
Not currently. I do not plan to add an alternative. `mongoc_collection_get_last_error` has had the deprecation attribute since 1.9.0. GitHub code search [shows no use of mongoc_collection_get_last_error](https://github.com/search?q=mongoc_collection_get_last_error+path%3A*.c+-repo%3Amongodb%2Fmongo-c-driver+NOT+is%3Afork+NOT+path%3A%2Fmongoc%5C-.*%5C.c%2F+NOT+path%3A%2Fbson%5C-.*%5C.c%2F+NOT+path%3Abson.c++NOT+path%3A%2Fmongocrypt%5C-.*%5C.c%2F+NOT+path%3A%2F.*test.*%2F&type=code&ref=advsearch), despite showing use of [mongoc_collection_remove](https://github.com/search?q=mongoc_collection_remove+%28+path%3A*.c+-repo%3Amongodb%2Fmongo-c-driver+NOT+is%3Afork+NOT+path%3A%2Fmongoc%5C-.*%5C.c%2F+NOT+path%3A%2Fbson%5C-.*%5C.c%2F+NOT+path%3Abson.c++NOT+path%3A%2Fmongocrypt%5C-.*%5C.c%2F+NOT+path%3A%2F.*test.*%2F&type=code&ref=advsearch), [mongoc_collection_insert](https://github.com/search?q=mongoc_collection_insert+%28+path%3A*.c+-repo%3Amongodb%2Fmongo-c-driver+NOT+is%3Afork+NOT+path%3A%2Fmongoc%5C-.*%5C.c%2F+NOT+path%3A%2Fbson%5C-.*%5C.c%2F+NOT+path%3Abson.c++NOT+path%3A%2Fmongocrypt%5C-.*%5C.c%2F+NOT+path%3A%2F.*test.*%2F&type=code&ref=advsearch), and [mongoc_collection_update](https://github.com/search?q=mongoc_collection_update+%28+path%3A*.c+-repo%3Amongodb%2Fmongo-c-driver+NOT+is%3Afork+NOT+path%3A%2Fmongoc%5C-.*%5C.c%2F+NOT+path%3A%2Fbson%5C-.*%5C.c%2F+NOT+path%3Abson.c++NOT+path%3A%2Fmongocrypt%5C-.*%5C.c%2F+NOT+path%3A%2F.*test.*%2F&type=code&ref=advsearch). So I expect this is non-disruptive. If there are reports requesting re-adding, a better named alternative could be considered (`mongoc_collection_get_last_write_result`?)

Reply:
Not necessarily a dedicated API to replace it, but is there a way to get the same information that this function provided?